### PR TITLE
fixed the gamma computation error

### PR DESCRIPTION
**Pull request by doyend**
The old code computes the sum of second-order differentiation, which is wrong. This one fixed it.

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by avolkov1** [comment]
Is the only code modification from:
```
drv = grad(loss_grads[0], inputs, torch.ones_like(loss_grads[0]) )
```
to:
```
drv = grad(loss_grads[0][0][2], inputs)
```

I'm looking at the function signature of grad:
``` torch.autograd.grad(outputs, inputs, grad_outputs=None, retain_graph=None, create_graph=False, only_inputs=True, allow_unused=False)
```

The change is very subtle. What are the components of `loss_grads`? It's not very clear what's contained in the loss_grads. So `loss_grads[0]` is the sum and `loss_grads[0][0][2]` is what then?

It looks good, but would be nice to clarify that pytorch operation.

**Comment by yidong72** [comment]
You can test it out by following code

```python
import torch
from torch.autograd import grad
'''
z = (xy)^2
x = 3, y =2

first order deriv [24 36]
d2z/dx2 = 8
d2z/dxdy = 24
d2z/dy2 = 18
'''

inputs = torch.tensor([3.0,2.0], requires_grad=True)

z = (inputs[0]*inputs[1])**2
first_order_grad = grad(z, inputs, create_graph=True)

second_order_grad_original, = grad(first_order_grad[0], inputs,
                                   torch.ones_like(first_order_grad[0]), retain_graph=True) # Does not give expected answer
second_order_grad_x, = grad(first_order_grad[0][0], inputs, retain_graph=True) #
second_order_grad_y, = grad(first_order_grad[0][1], inputs)
```
the old code is to calcuate the sum of gradients with respect to the parameters, i.e. (d2z/dx2 + d2z/dxdy, d2z/dy2 + d2z/dxdy)

**Comment by yidong72** [comment]
I added the example into the notebook
